server/network.js

```diff
@@ -470,9 +470,11 @@ network.listNetwork_ = function(networkPath) {
  */
 network.incrementalEdges_ = function(file, genes, nodes) {
   var oldNodes = {};
-  nodes.forEach(function(node) {
-    oldNodes[node.id] = true;
-  });
+  if (nodes.length) {
```

Comment:
What error is it? I don't get it... If nodes = [], the the forEach has no effect, so that oldNodes is an empty object as it is, and everything will be the same?